.github/workflows/_win-test.yml

```diff
@@ -63,6 +63,21 @@ jobs:
         run: |
           git config --global core.symlinks true
 
+      - name: Setup SSH (Click me for login details)
+        uses: pytorch/test-infra/.github/actions/setup-ssh@main
+        with:
+          github-secret: ${{ secrets.GITHUB_TOKEN }}
+          instructions: |
```

Comment:
#til that we added support for this :D

Reply:
Was about to comment, that you've reviewed the change that enabled it, but then check and indeed you didn't :P